awesomemotive/one-click-demo-import
# Updates npm dev dependencies

## Description

This PR updates the npm dev dependencies.

## Motivation

Resolves #279 and the following dependabot PRs:
- #275
- #269
- #268 
- #264
- #253
- #249
- #246
- #243
- #242

## Key Changes

The changes require `Node v18` and we also added the npm package [sass](https://www.npmjs.com/package/sass) because the latest version `gulp-sass` requires it.

## Testing procedure

1. Make sure you're running `Node v18`. If you're using MacOS, you can use [nvm](https://github.com/nvm-sh/nvm) to easily change node versions.
2. Delete `node_modules` directory.
3. In the repo's root, run `npm install`.
4. Then run `npm run build`.
5. You should be able to get the build files.

If you encounter errors during `npm install`, you might have to install python2. I encountered this issue because I have python3 installed in my system.
